### DISCUSSION TITLE
Request: More speed options

### DISCUSSION THREAD

**Discussion by Plekteret** [TortugaPower/BookPlayer#585] [Ideas]
Hey there! First of all, thank you so much for creating this app. It is such a life saver, and I absolutely love the ios 14 widget that comes with it as well! 

As for my request, I was wondering if it would be possible to get more speed intervals between 1x and 2x, like for example 1.2x,1.3x,1.4x etc, as the increments today can be quite steep. 

I hope you consider it in the future and again, thank you for such a wonderful app!

**Comment by GianniCarlo** [comment]
Hi @Plekteret! this is something that we're working on, the approach we are taking, is replacing the current control, with one that lets you step in +/- 0.1 intervals, which will address most use cases when it comes to speed requests 👍

**Reply by Plekteret** [reply]
Oh that sounds utterly amazing! I can't wait for it, thank you for the swift reply <3
